app/api/helpers/custom_fields.py

```diff
@@ -245,3 +246,15 @@ def validate(self, value):
             self.validation_error = 'Value of %s is not in available choices'
             return False
         return True
+
+
+class Licence(CustomField):
+    """
+    Custom field that gets the licence details given the licence name
+    """
+    __schema_example__ = "{'name': 'Licence name' 'long_name': 'Long licence name', \
```

Comment:
@shubham-padia Can you please explain me about #3238 which this PR fixes.
The title is "Wrong Information in REST API for FOSSASIA 2017 Summit" so it can be fixed by just updating the copyright field of the event, why is there a need to add a new field called "license" here?